packages/connectivity/src/components/ConnectivityWebhooks/ConnectivityWebhooksForm.tsx

```diff
@@ -46,6 +48,13 @@ export const ConnectivityWebhooksForm: FC<IConnectivityWebhooksForm> = ({ data }
     };
   }, [dispatch]);
 
+  const toggleTestDialog = useCallback(() => {
+    if (openTestDialog) {
+      dispatch(connectivityActions.cleanWebhookTestMessage());
+    }
+    setOpenTestDialog(!openTestDialog);
+  }, [setOpenTestDialog, openTestDialog]);
```

Comment:
add dispatch to useCallback dependencies. no need to add connectivity to deps because it's imported from other file